mslice/plotting/pyplot.py

```diff
@@ -2603,7 +2624,7 @@ def loglog(*args, **kwargs):
 
 # This function was autogenerated by boilerplate.py.  Do not edit as
 # changes will be lost
-@_autogen_docstring(Axes.magnitude_spectrum)
+#@_autogen_docstring(Axes.magnitude_spectrum)
```

Comment:
@mducle We'll probably need to wrap it with our own definition and check internally on the version: http://stackoverflow.com/questions/10724854/how-to-do-a-conditional-decorator-in-python-2-6